tests/react/vanilla-utils/atomWithObservable.test.tsx

```diff
@@ -2,17 +2,13 @@ import { Component, StrictMode, Suspense, useState } from 'react'
 import type { ReactElement, ReactNode } from 'react'
 import { act, fireEvent, render, waitFor } from '@testing-library/react'
 import { BehaviorSubject, Observable, Subject, delay, of } from 'rxjs'
-import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+import { afterEach, describe, expect, it, vi } from 'vitest'
 import { fromValue, makeSubject, pipe, toObservable } from 'wonka'
 import { useAtom, useAtomValue, useSetAtom } from 'jotai/react'
 import { atom, createStore } from 'jotai/vanilla'
 import { atomWithObservable } from 'jotai/vanilla/utils'
 
-beforeEach(() => {
-  vi.useFakeTimers({ shouldAdvanceTime: true })
-})
```

Comment:
Should we revert to use `beforeEach` or leave it as is?

Reply:
Yup!! I reverted it in [this commit](https://github.com/pmndrs/jotai/pull/1913/commits/bf76d1c255c473760a76e94e83fa00168c559a6b)